docs/design.ltx

```diff
@@ -0,0 +1,337 @@
+
+Each section was authored by the corresponding team member responsible for each component identified in the technology review: Sections \ref{subsec:featureextraction} and \ref{subsec:classification} by Heidi; section \ref{subsec:identifymining} by Xiaomei; sections \ref{subsec:preprocessing}, \ref{subsec:correlate}, and \ref{subsec:changesovertime} by Taylor; section \ref{subsec:apidoc} by Heidi; and sections \ref{subsec:staticsitegenerator} and \ref{subsec:frontendframework} by Xiaomei. The research priorities and design choices for each part of the project are discussed in each section.
+
+\section{Technologies}
+
+% heidi: requirement 3.4.1 part 1: choose technology for mineral identification and classification
+\subsection{Feature extraction in AVIRIS data}
+\label{subsec:featureextraction}
+
+In order to process hyperspectral data, machine learning techniques will have to be used to process the raw AVIRIS images \cite{aviris}. Further processing could be done using blob detection, as described below.
+
+AVIRIS images cover a lot of land mass with a lot of different features, so a method of feature extraction is needed to automatically pick out the different regions on the image. In blob detection, an area of an image that is different from the surrounding pixels is known as a blob. This will make classification easier as instead of needing to classify every single pixel every single time, the classification algorithm will only need to process each blob, or “feature,” which typically contains many pixels. The blob detection algorithm that will be used is automatic scale detection. It is one of the more expensive methods of blob detection, but it is highly accurate, so it is the one that has been chosen as discussed in the technology review. The automatic scale detection algorithm is based on the Laplacian of Gaussian. As the name suggests, this is a convolution of the Laplacian and Gaussian filters \cite{autoscaleprinc}. The Gaussian filter is applied first to smooth out noise and then the Laplacian filter is used to detect the shapes in the image \cite{lapofgaus}. The equation for a Gaussian convolution is given below, where t represents scale \cite{autoscaleprinc}.
+
+\[ g(x, y) = \frac{1}{2\pi t}e^{-\frac{(x^{2} + y^{2})}{2t}}\] 
+
+The equation for a Laplacian filter is given below, where t represents scale \cite{autoscaleprinc}.
+
+
+\[\Delta^{2}L = t(L_{xx} + L_{yy}) \]
+
+
+When the Laplacian filter is applied to the Gaussian filter, the equation becomes the one shown below \cite{lapofgaus}.
+
+\[LoG(x, y) = -\frac{1}{\pi t^{4}}[1 - \frac{x^{2} + y^{2}}{2t}] e^{\frac{-x^{2} + y^{2}}{2t^{2}}}\]
```

Comment:
I asked the same question when reviewing the section. Apparently "Laplacian of Gaussian" is jargon for a [smoothed spatial derivative](http://homepages.inf.ed.ac.uk/rbf/HIPR2/log.htm) function used in blob detection.